### PR TITLE
Update document for new changes

### PR DESCRIPTION
**Pull request by yoonghm**

### CODE REVIEW

**Comment by yoonghm** [comment]
Reformat documentation to reflect the latest code.

File: docs/commands.md [hunk 1]
```diff
@@ -2,22 +2,22 @@
 
 ## Create
 
-### TS.CREATE
+### `TS.CREATE`
```
Comment (K-Jo) [inline docs/commands.md]: If we want to keep this in line with other modules, please remove the backticks

File: docs/commands.md [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
 
 Create a new time-series.
 
 ```sql
-TS.CREATE key [RETENTION retentionSecs] [LABELS field value..]
+TS.CREATE key [RETENTION retentionSecs] [CHUNK_SIZE chunksize] [LABLES <labelName labelValue> ...]
```
Comment (K-Jo) [inline docs/commands.md]: we chose not to publish our chunksize for now, please rmove

Comment (K-Jo) [inline docs/commands.md]: can you remove the `<` and `>` and spell LABELS correct?

File: docs/commands.md [hunk 3]
```diff
@@ -2,22 +2,22 @@
...
-* key - Key name for timeseries
+* `key` - Key name for a timeseries
 
 Optional args:
 
- * retentionSecs - Maximum age for samples compared to current time (in seconds)
-    * Default: The global retenion secs configuration of the database (by default, `0`)
-    * When set to 0, the series is not trimmed at all
- * labels - Set of key-value pairs that represent metadata labels of the key
+ * `retentionSecs` - Maximum age for samples compared to current time (in seconds)
```
Comment (K-Jo) [inline docs/commands.md]: in this change it's no longer clear that there is a global module specific default.  that if even not set is 0

File: docs/commands.md [hunk 4]
```diff
@@ -27,23 +27,23 @@ TS.CREATE temperature RETENTION 60 LABELS sensor_id 2 area_id 32
 
 ## Update
 
-### TS.ADD
+### `TS.ADD`
```
Comment (K-Jo) [inline docs/commands.md]: no ``

File: docs/commands.md [hunk 5]
```diff
@@ -27,23 +27,23 @@ TS.CREATE temperature RETENTION 60 LABELS sensor_id 2 area_id 32
...
 
 Append (or create and append) a new value to the series.
 
 ```sql
-TS.ADD key timestamp value [RETENTION retentionSecs] [LABELS field value..]
+TS.ADD key timestamp value [RETENTION retentionSecs] [LABLES <labelName labelValue> ...]
```
Comment (K-Jo) [inline docs/commands.md]: labELS <>

File: docs/commands.md [hunk 6]
```diff
@@ -27,23 +27,23 @@ TS.CREATE temperature RETENTION 60 LABELS sensor_id 2 area_id 32
...
+* `value` - Sample numeric data value (double)
 
-These arguments are optional because they can be set by TS.CREATE:
+These arguments are optional because they can be set by `TS.CREATE`:
 
- * retentionSecs - Maximum age for samples compared to current time (in seconds)
-    * Default: The global retenion secs configuration of the database (by default, `0`)
-    * When set to 0, the series is not trimmed at all
- * labels - Set of key-value pairs that represent metadata labels of the key
+ * `retentionSecs` - Maximum age for samples compared to current time (in seconds)
```
Comment (K-Jo) [inline docs/commands.md]: same default comment

File: docs/commands.md [hunk 7]
```diff
@@ -63,37 +63,31 @@ The complexity of `TS.ADD` is always O(M) when M is the amount of compactions ru
 
 - You can use this command to add data to an non existing timeseries in a single command.
   This is the reason why `labels` and `retentionsecs` are optional arguments.
-- When specified and the key doesn't exist, RedisTimeSeries will create the key with the specified `labels` and or `retentionSecs`.
-  Setting the `labels` and `retentionSecs` introduces additional time complexity.
+- When specified and the key doesn't exist, RedisTimeSeries will create the key with the specified `labels` and or `retentionSecs`.  
+  Setting the `labelName`, `labelValue` and `retentionSecs` introduces additional time complexity.
 
-### TS.INCRBY/TS.DECRBY
+### `TS.INCRBY`/`TS.DECRBY`
```
Comment (K-Jo) [inline docs/commands.md]: ``

File: docs/commands.md [hunk 8]
```diff
@@ -63,37 +63,31 @@ The complexity of `TS.ADD` is always O(M) when M is the amount of compactions ru
...
+* `value` - Sample numeric data value (double)
 
 Optional args:
 
-* time-bucket - Time bucket for resetting the current counter in seconds
-* retentionSecs - Maximum age for samples compared to current time (in seconds)
-  * Default: The global retenion secs configuration of the database (by default, `0`)
-  * When set to 0, the series is not trimmed at all
-* labels - Set of key-value pairs that represent metadata labels of the key
+* `timeBucket` - Time bucket for resetting the current counter in seconds
```
Comment (K-Jo) [inline docs/commands.md]: same default comment

File: docs/commands.md [hunk 9]
```diff
@@ -104,60 +98,60 @@ If this command is used to add data to an existing timeseries, `retentionSecs` a
 
 ## Aggregation, Compaction, Downsampling
 
-### TS.CREATERULE
+### `TS.CREATERULE`
```
Comment (K-Jo) [inline docs/commands.md]: ``

File: docs/commands.md [hunk 10]
```diff
@@ -104,60 +98,60 @@ If this command is used to add data to an existing timeseries, `retentionSecs` a
...
+* `sourcekey` - Key name for source timeseries
+* `destKey` - Key name for destination time series
+* `aggregationType` - Aggregation type: `avg`, `sum`, `min`, `max`, `range`, `count`, `first`, `last`
+* `bucketSizeSeconds` - Time bucket for aggregation in seconds
 
-DEST_KEY should be of a `timeseries` type, and should be created before TS.CREATERULE is called.
+`destKey` should be of a `timeseries` type, and should be created before `TS.CREATERULE` is called.
 
-### TS.DELETERULE
+### `TS.DELETERULE`
```
Comment (K-Jo) [inline docs/commands.md]: ``

File: docs/commands.md [hunk 11]
```diff
@@ -104,60 +98,60 @@ If this command is used to add data to an existing timeseries, `retentionSecs` a
...
 
-- sourceKey - Key name for source time series
-- destKey - Key name for destination time series
+* `sourcekey` - Key name for source timeseries
+* `destKey` - Key name for destination time series
 
 ## Query
 
-### TS.RANGE
+### `TS.RANGE`
```
Comment (K-Jo) [inline docs/commands.md]: ``

File: docs/commands.md [hunk 12]
```diff
@@ -180,22 +174,22 @@ But because m is pretty small, we can neglect it and look at the operation as O(
    2) "20"
 ```
 
-### TS.MRANGE
+### `TS.MRANGE`
```
Comment (K-Jo) [inline docs/commands.md]: ``

File: docs/commands.md [hunk 13]
```diff
@@ -241,15 +235,15 @@ Optional args:
          2) "20"
 ```
 
-### TS.GET
+### `TS.GET`
```
Comment (K-Jo) [inline docs/commands.md]: ``

File: docs/commands.md [hunk 14]
```diff
@@ -261,15 +255,15 @@ TS.GET key
 
 ## General
 
-### TS.INFO
+### `TS.INFO`
```
Comment (K-Jo) [inline docs/commands.md]: ``

**Pull request closed without merging**